sympy/ntheory/tests/test_digits.py

```diff
@@ -8,10 +8,10 @@ def test_digits():
                 for n in range(20)])
     assert all([digits(n, 16)[1:] == [int(d, 16) for d in format(n, 'x')]
                 for n in range(20)])
-    assert digits(2345, 34) == [34, 2, 0, 33]
-    assert digits(384753, 71) == [71, 1, 5, 23, 4]
-    assert digits(93409) == [10, 9, 3, 4, 0, 9]
-    assert digits(-92838, 11) == [-11, 6, 3, 8, 2, 9]
+    assert digits(2345, 34, None) == [34, 2, 0, 33]
+    assert digits(384753, 71, None) == [71, 1, 5, 23, 4]
+    assert digits(93409, 10, None) == [10, 9, 3, 4, 0, 9]
+    assert digits(-92838, 11, None) == [-11, 6, 3, 8, 2, 9]
```

Comment:
```suggestion
    assert digits(2345, 34) == [34, 2, 0, 33]
    assert digits(384753, 71) == [71, 1, 5, 23, 4]
    assert digits(93409, 10) == [10, 9, 3, 4, 0, 9]
    assert digits(-92838, 11) == [-11, 6, 3, 8, 2, 9]
```
`None` isn't needed for `bits`, it's the default value for that argument so given or not, doesn't make a difference. 
Can you test it for some value of `bits` which is not `None` like, 2, 3 or anything simple?

Reply:
Okay sir. Working on it.